src/platform/backends/lxd/lxd_vm_image_vault.cpp

```diff
@@ -225,13 +195,64 @@ bool mp::LXDVMImageVault::has_record_for(const std::string& name)
 
 void mp::LXDVMImageVault::prune_expired_images()
 {
-    mpl::log(mpl::Level::trace, category, "Pruning expired images not implemented");
+    auto json_reply = lxd_request(manager, "GET", QUrl(QString("%1/images").arg(base_url.toString())));
+
+    auto images = json_reply["metadata"].toArray();
+
+    for (const auto image : images)
+    {
+        auto image_info = image.toObject();
+        auto last_used_count =
+            QDateTime::fromString(image_info["last_used_at"].toString(), Qt::ISODateWithMs).toMSecsSinceEpoch();
```

Comment:
It is just breaking down info into smaller chunks for readability and is used in the next line.  It is just like this in `default_vm_image_vault` as well.

Reply:
I just mean the variable name… it's a timestamp, not a count?

Reply:
Well, it is a count of milliseconds since the epoch and is the same variable name used in `default_vm_image_vault`.  If you prefer it be named something else, I'm fine with that.

Reply:
`last_used_msecs`?

Reply:
Sure, I'm fine with that.  Should I change this variable name in `DefaultVMImageVault` as well since that is the inspiration for this original name?

Reply:
I actually just got rid of this temporary variable and combined it all with the line below it.